target/dovecot/dovecot-ldap.conf.ext

```diff
@@ -2,7 +2,7 @@ base                = ou=people,dc=domain,dc=com
 default_pass_scheme = SSHA
 dn                  = cn=admin,dc=domain,dc=com
 dnpass              = admin
-hosts               = mail.domain.com
+uris                = mail.domain.com
```

Comment:
Doesn't this need a protocol then?

Reply:
Yes, ihmo this is missing the protocol
https://doc.dovecot.org/configuration_manual/authentication/ldap/

Reply:
Yes, it should be `ldaps://mail.domain.com` - it's replaced though by the config script, so it shouldn't influence functionality.

Reply:
I have fixed that now, see https://github.com/docker-mailserver/docker-mailserver/compare/6ac6c241039efd1a22c5809a5434978d37585e94..0a3ab8767f7bddc0a4491494d95cec1173a8d9f2